packages/format-library/package.json

```diff
@@ -31,6 +31,7 @@
 		"@wordpress/block-editor": "file:../block-editor",
 		"@wordpress/components": "file:../components",
 		"@wordpress/compose": "file:../compose",
+		"@wordpress/core-data": "^7.0.0",
```

Comment:
This should be changed to

```
"@wordpress/core-data": "file:../core-data",
```

so it's in line with how it's done in the `package.json`

Reply:
Done!